br/pkg/streamhelper/advancer.go

```diff
@@ -424,8 +443,12 @@ func (c *CheckpointAdvancer) onTaskEvent(ctx context.Context, e TaskEvent) error
 		c.task = e.Info
 		c.taskRange = spans.Collapse(len(e.Ranges), func(i int) kv.KeyRange { return e.Ranges[i] })
 		c.setCheckpoints(spans.Sorted(spans.NewFullWith(e.Ranges, 0)))
-		c.lastCheckpoint = newCheckpointWithTS(e.Info.StartTs)
-		p, err := c.env.BlockGCUntil(ctx, c.task.StartTs)
+		globalCheckpointTs, err := c.GetGlobalCheckpointWithRetry(ctx, 3, 100*time.Millisecond)
```

Comment:
How is 3*100ms determined for retry duration?

Reply:
Just arbitrary，maybe we need a exponential backoff time?

Reply:
I will preferring directly return this error, we may add a test case to verify this case.